components/Card.vue

```diff
@@ -17,13 +17,19 @@ export default {
       type:    String,
       default: 'go'
     },
+  },
+
+  methods: {
+    hasSlot(name) {
+      return !!this.$slots[name] || !!this.$scopedSlots[name];
+    }
```

Comment:
You can't judge with hasSlot，  Users may want to add a title through prop

Reply:
Yes, i will improve the conditions, thanks!